curriculum/challenges/english/25-front-end-development/workshop-binary-search/683ec95f2f8781355556ff84.md

```diff
@@ -0,0 +1,55 @@
+---
+id: 683ec95f2f8781355556ff84
+title: Step 12
+challengeType: 20
+dashedName: step-12
+---
+
+# --description--
+
+If the condition in the `elif` is true, then remove the `pass` keyword and update the value of the `low` variable by adding `1` to the `mid` variable.
+
+This will extend the search to the right half of the current search areas in the list, because if the `value` is greater than `value_at_middle`, it means the `value` must be in the right half of the current search area.
+
+# --hints--
+
+You should remove the `pass` keyword and update the `low` variable to `mid + 1`.
+
+```js
+({
+    test: () =>
+        assert(
+            runPython(
```

Comment:
It's not a good idea to check things this way. If someone adds a `print` call somewhere, the test will fail and they won't know why. You can go directly to the `elif` with `_Node(_code).find_function("binary_search").find_whiles()[0].find_ifs()[0].find_bodies()[1]` and check if there's what you want with `has_stmt()` or `is_ordered()` depending on the case.